### ISSUE TITLE
SignInUp Modal misalignment for devices smaller than 400px width

### ISSUE THREAD

**Issue by ehconitin** [twentyhq/twenty#6385]
## Bug Description

Signinup form is misalligned for devices smaller than 400px width

**Example:** 

https://github.com/user-attachments/assets/d03b79d0-954b-49bd-bc37-040997147716

## Expected behavior

SignInUp modal should always allign perfectly in center

**Comment by ehconitin** [comment]
@Bonapara 

Created a [PR 6385](https://github.com/twentyhq/twenty/pull/6386)

Please let me know what do you think.

**Comment by Bonapara** [comment]
Thanks for raising both the issue and the PR @ehconitin ;) Very good point! Did you see [this issue](https://github.com/twentyhq/twenty/issues/5740)? I wonder if it could not be handled at the same time

**Comment by ehconitin** [comment]
Will have a look into it.

**Comment by ehconitin** [comment]
@Bonapara 

SignInUp screen could be easily made fullscreen. But I am having trouble rendering pricing screen, the server returns error whenever I hit it, something on the lines of `could not found base-plan`.

**Comment by Bonapara** [comment]
maybe @martmull will have an idea!

**Comment by martmull** [comment]
Yeah, you need a stripe instance with a base-plan product to display the pricing screen properly

**Comment by ehconitin** [comment]
Is there any documentation on how do i setup stripe instance for this project? @martmull

**Comment by martmull** [comment]
No,actually this is a feature we only need for our self hosted instance so no doc. What you need is those env variables:
```
IS_BILLING_ENABLED=true
BILLING_STRIPE_WEBHOOK_SECRET=any_value
BILLING_STRIPE_API_KEY='rk_test_<...>'
BILLING_STRIPE_BASE_PLAN_PRODUCT_ID='prod_<...>'
```
I have created you an access to our test Stripe instance, I have sent you a friend request on Discord (username: `neeko2853`) to give you your API_KEY and product_id

**Comment by ehconitin** [comment]
Thats the correct username, have accepted your request.
Thanks for the access!🫡

**Comment by martmull** [comment]
thank you @ehconitin, just t say I have deleted the stripe key

**Comment by ehconitin** [comment]
@martmull okay! Thanks 🙂
Could I ask for it again if I need it in some issue?

**Comment by martmull** [comment]
Yes of course !
